chain-tx-enclave-next/tdbe/enclave-app/src/sgx_module/handler.rs

```diff
@@ -0,0 +1,16 @@
+use chain_core::tx::data::input::TxoPointer;
+use chain_core::tx::data::TxId;
+
+/// Retrieves all the spent transactions for given transaction IDs
+///
+/// Note: This is a dummy implementation. FIXME: implement
+pub fn get_spent_transaction_outputs(_txids: Vec<TxId>) -> Result<Vec<TxoPointer>, String> {
+    Ok(Vec::new())
+}
```

Comment:
this is kind of "minimal" thing that should have been implemented in order to fix #1549

Reply:
for the server / responding side, it can reuse tx-query parts.

for the client, what to request could be e.g. provided as the command line params temporarily?

Reply:
> for the server / responding side, it can reuse tx-query parts.

What do you mean by reusing tx-query parts?

> for the client, what to request could be e.g. provided as the command line params temporarily?

Currently, we cannot pass any command line arguments to enclave except for IP addresses.

Reply:
> What do you mean by reusing tx-query parts?

fetching sealed payloads from chain-abci storage via zmq?

> Currently, we cannot pass any command line arguments to enclave except for IP addresses.
it could connect to some `init_params` hostname where the usercall extension will dump fixed provided transaction IDs?